internal/event/dispatch_service.go

```diff
@@ -31,50 +32,28 @@ type Event interface {
 
 type Handler func(event Event)
 
-type controlType int
-
-const (
-	subscribe controlType = iota
-	subscribeSync
-	unsubscribe
-)
-
 const (
 	created int32 = iota
```

Comment:
Do we really need 3 states ? I think we can eliminate one of them. `created` and `ready` seems like same thing to me.

Reply:
Removed at: https://github.com/hazelcast/hazelcast-go-client/pull/644/commits/77bff36b7a0d762a4af5b25150d7d694c9d8028f